docs/content/formatting.stories.mdx

```diff
@@ -161,3 +161,12 @@ abbreviating numbers at ten thousand when precision is not crucial.
 | --------- | ------------ |
 | $3,345    | $3,345.34    |
 | $32k      | $32,462      |
+
+Examples of essential decimal places include payment-related details, accounting
+statements, measurements, and reporting tables where exact amounts may be
+required.
+
+| ✅ **Do**                      | ❌ **Don't**             |
+| ------------------------------ | ------------------------ |
+| Balance: **$489.62**           | Balance: **$489**        |
+| Received payment for $5,240.18 | Received payment for $5k |
```

Comment:
LGTM!
What would a measurement example be?

Your house is 4.815x smaller than @andy-j 's house?

Reply:
@chris-hut I think my house is actually 5.23× smaller than Andy's house